src/AutomatedCar/Models/ExternalGearbox.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace AutomatedCar.Models
+{
+    using Avalonia.Media;
+    using ReactiveUI;
+    using System;
+    using SystemComponents;
+
+    public class ExternalGearbox : ReactiveObject
+    {
+        private int currentExternalGear = 0;  //0: P, 1: R, 2: N, 3: D
```

Comment:
You could use an enum with explicitly defined indexes, instead of numbers.
```
        public enum Gear
        {
            P = 0, R = 1, N = 2, D = 3
        }
        ...
        private Gear currentExternalGear = Gear.P;
```
This would make the gear handling below a bit cleaner and more understandable.